pkg/proxy/ipvs/proxier.go

```diff
@@ -753,19 +736,6 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 // return an error if it fails to get the kernel modules information without error, in which
 // case it will also return false.
 func CanUseIPVSProxier(handle KernelHandler, ipsetver IPSetVersioner) (bool, error) {
-	mods, err := handle.GetModules()
-	if err != nil {
-		return false, fmt.Errorf("error getting installed ipvs required kernel modules: %v", err)
-	}
-	wantModules := sets.NewString()
-	loadModules := sets.NewString()
-	wantModules.Insert(ipvsModules...)
-	loadModules.Insert(mods...)
-	modules := wantModules.Difference(loadModules).UnsortedList()
```

Comment:
I think we still need check whether kernel modules are loaded in kube-proxy, right?

Reply:
i keep it now,  will deleting it once all function well in `kubeadm`.